File: app/src/main/java/com/walhalla/landing/activity/WPresenter.java
package com.walhalla.landing.activity;

import android.content.Intent;


import com.walhalla.webview.ChromeView;
import com.walhalla.webview.UWView;

public interface WPresenter {


    void onActivityResult(int requestCode, int resultCode, Intent data);

    void a123(ChromeView chromeView, UWView webView
    //, RelativeLayout child, RelativeLayout browser
     );

    void onConfirmation__(boolean allowed, String[] resources);
}

File: app/src/main/java/net/walhalla/landing/activity/WPresenter.kt
package net.walhalla.landing.activity

import android.content.Intent
import androidx.appcompat.UWView
import com.walhalla.webview.ChromeView

interface WPresenter {
    fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?)

    fun a123(
        chromeView: ChromeView?, webView: UWView? //, RelativeLayout child, RelativeLayout browser
    )

    fun onConfirmation__(allowed: Boolean, resources: Array<String?>?)
}
